client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -1,57 +1,90 @@
 import { CountComparisonCard } from '@automattic/components';
+import { UseQueryResult } from '@tanstack/react-query';
 import { translate } from 'i18n-calypso';
 import React from 'react';
+import useSubscribersQuery from 'calypso/my-sites/stats/hooks/use-subscribers-query';
 
-function SubscribersOverviewCardStats() {
-	const overviewCardStats = [
-		{
-			heading: translate( 'Today' ),
-			count: 1410,
-		},
-		{
-			heading: translate( '30 days ago' ),
-			count: 1080,
-			previousCount: 1080,
-		},
-		{
-			heading: translate( '60 days ago' ),
-			count: 1000,
-			previousCount: 1080,
-		},
-		{
-			heading: translate( '90 days ago' ),
-			count: 970,
-			previousCount: 1080,
-		},
-	];
-	return overviewCardStats;
+interface SubscribersData {
+	period: string;
+	subscribers: number;
+	subscribers_change: number;
 }
 
-function SubscribersOverviewCards() {
-	const overviewCardStats = SubscribersOverviewCardStats();
+interface SubscribersDataResult {
```

Comment:
Those two definitions are also defined in `subscribers-section`. Let's avoid duplication. We can either export it from there and import it here, or a better option would be to keep it with the hook because more places will use it.
You could either convert the hook to TS and export it from there (this would be best) or create a file with types and place it in the hook directory.